### DISCUSSION TITLE
How do you do different colors in text controls?

### DISCUSSION THREAD

**Discussion by ericjlima** [storybookjs/storybook#12813] [Help] [answered]
Say you have text that looks like this: 
![image](https://user-images.githubusercontent.com/3118306/96372918-5745d580-1137-11eb-9460-8bd06b32f3b0.png)

How do you go about adding a span to change this color in the control element of storybook? It only allows for string texts and not any actual elements. 

```

{
      control: {
        type: 'text',
        options: ['Find the right fit.', 'Lorem Ipsum'],
      },
      name: 'title',
      description: 'Change title',
      defaultValue: 'Find the <span>right</span> fit.',
      table: {
        defaultValue: { summary: 'Lorem Ipsum' },
      },
    }
```

**Comment by greg-skorupka** [comment, marked as answer]
`defaultValue: <>Find the <span>right</span> fit.</>,` works I believe to render the span correctly but not so much if try to enter/type your own custom code in storybook

Yeah this one is tricky for a CMS

**Reply by ericjlima** [reply]
this works. Thanks Greg :)

**Comment by greg-skorupka** [comment]
maybe `escapeHTML: false` would help?
https://github.com/storybookjs/storybook/issues/8635
